lms/static/sass/features/_unsupported-browser-alert.scss

```diff
@@ -0,0 +1,51 @@
+
+@import 'lms/theme/variables-v1';
+
+// Unsupported browser alert
+$browser-update-background: #fff !important;
+$browser-update-ok-button: $button-color;
+$browser-update-ignore-button: $action-primary-disabled-bg;
+
+.buorg {
```

Comment:
Is this the name from the third party library you are using?  It is a confusing name, if it cannot be renamed can we at least add some clarification in comments?